BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -348,7 +351,9 @@ public Part GetPart()
                     if (weapon.Current.GetShortName() != GetShortName()) continue;
                     if (weapon.Current.AmmoID != AmmoID && weapon.Current.AmmoID != lastAmmoID)
                     {
-                        vessel.GetConnectedResourceTotals(weapon.Current.AmmoID, out double ammoCurrent, out double ammoMax);
+                        double ammoCurrent;
```

Comment:
The variable declarations can be inlined for cleaner code as
```csharp
GetAmmoCount(out double ammoCurrent, out double ammoMax);
```

Reply:
Sorry, that was something left behind from an earlier change that I forgot to switch back.